app/lib/components/term.js

```diff
@@ -11,6 +32,18 @@ export default class Term extends Component {
     this.onWheel = this.onWheel.bind(this);
     this.onScrollEnter = this.onScrollEnter.bind(this);
     this.onScrollLeave = this.onScrollLeave.bind(this);
+
+    for (const entry in actions) {
```

Comment:
I'd like to avoid this sort of meta-programming since it'll have a negative effect on startup time and static analysis.